plotter/polygon.go

```diff
@@ -65,7 +65,7 @@ func (pts *Polygon) Plot(c draw.Canvas, plt *plot.Plot) {
 	}
 	if pts.Color != nil && len(ps) > 0 {
 		c.SetColor(pts.Color)
-		var pa vg.Path
+		pa := make(vg.Path, 0, 4*len(ps))
```

Comment:
Can we have a comment for why this is `4*len(ps)`. It's not immediately obvious from the context.

Reply:
ok.
also note that if I am not mistaken, the `for-range` below should probably range over `ring[1:]` instead of just `ring`. @ctessum do you agree?
(applying this modification, only the `svg` and `eps` tests fail)

Reply:
Yes, that seems right.

Reply:
done.